rocky/crisis_room/templates/partials/new_dashboard_item_modal.html

```diff
@@ -0,0 +1,44 @@
+{% load i18n %}
+
+{% component "modal" modal_id="new-dashboard-item-modal" size="dialog-medium" %}
+{% fill "header" %}
+{% blocktranslate with item_type=item_type %}
+            Add {{ item_type }} to dashboard
+        {% endblocktranslate %}
+{% endfill %}
+{% fill "content" %}
+<p>
+    {% blocktranslate with item_type=item_type %}
+        Add these {{ item_type }} with the selected filters
+        to the dashboard of your choice.
+    {% endblocktranslate %}
+</p>
+<form method="post" id="new-dashboard-item-form">
+    {% csrf_token %}
+    {% include "partials/form/form_errors.html" with form=object_list_settings_form %}
+    {% include "forms/new_dashboard_item_form_fields.html" %}
+    {% include "partials/form/fieldset.html" with fields=object_list_settings_form form_view="horizontal" %}
+
+    <label>
+        {% translate "Show table columns" %}
+        <span class="nota-bene" aria-hidden="">({% translate "Required" %})</span>
+    </label>
+    {% for key, column_name in table_columns.items %}
+        <div class="checkbox">
+            <input type="checkbox" name="column_values" value="{{ key }}" checked>
+            <input type="hidden" name="column_names" value="{{ column_name }}">
+            <label>{{ column_name }}</label>
```

Comment:
```suggestion
            <input type="checkbox" name="column_values" value="{{ key }}" checked id="column_{{key}}">
            <input type="hidden" name="column_names" value="{{ column_name }}">
            <label for="column_{{key}}">{{ column_name }}</label>
```

Reply:
This would make the column names (in this dropdown) clickable and trigger their corresponding checkbox.